docs/UserGuide.adoc

```diff
@@ -207,41 +207,66 @@ exp/23-Nov-2020 a/21`
 
 ==== Add with system prompting
 
-In this mode, the user can add the person's attributes one by one.
-After entering an attribute, the system will prompt the user input for another attribute.
+In this mode, instead of typing a long command, you can add a person's attributes one by one.
+After entering an attribute, the system will prompt you to input another attribute.
 
 Format: `add t/**TYPE**`
```

Comment:
Perhaps you can change this to:
To add a person with this mode, enter the following command of the format `add t/**TYPE**`.